### ISSUE TITLE
Toolbar.php :: ErrorException: Array to string conversion

### ISSUE THREAD

**Issue by matahr** [bcit-ci/CodeIgniter4#254]
```
2016/09/08 17:58:59 [error] 3545#3545: *1164 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Method CodeIgniter\HTTP\Header::__toString() must not throw an exception, caught ErrorException: Array to string conversion in /CodeIgniter/application/Views/errors/html/error_exception.php on line 270
PHP message: PHP Stack trace:
PHP message: PHP   1. CodeIgniter\Debug\Exceptions->exceptionHandler($exception = *uninitialized*) /CodeIgniter/system/Debug/Exceptions.php:0
PHP message: PHP   2. include() /CodeIgniter/system/Debug/Exceptions.php:150
PHP message: PHP   3. natsort(*uninitialized*) /CodeIgniter/application/Views/errors/html/error_exception.php:270
PHP message: PHP Fatal error:  Method CodeIgniter\HTTP\Header::__toString() must not throw an exception, caught ErrorException: Array to string conversion in /CodeIgniter/application/Views/errors/html/error_exception.php on line 270
PHP message: PHP Stack trace:
PHP message: PHP   1. CodeIgniter\Debug\Exceptions->exceptionHandler($exception = *uninitialized*) /CodeIgniter/system/Debug/Exceptions.php:0
PHP message: PHP   2. include() /CodeIgniter/system/Debug/Exceptions.php:150
PHP message: PHP   3. natsort(*uninitialized*) /CodeIgniter/application/Views/errors/html/error_exception.php:270
PHP message: PHP   4. CodeIgniter\Debug\Exceptions->shutdownHandler() /CodeIgniter/system/Debug/Exceptions.php:0
PHP message: PHP   5. CodeIgniter\Debug\Exceptions->exceptionHandler($exception = *uninitialized*) /CodeIgniter/system/Debug/Exceptions.php:199
PHP message: PHP   6. include() /CodeIgniter/system/Debug/Exceptions.php:150
PHP message: PHP   7. natsort(*uninitialized*) /CodeIgniter/application/Views/errors/html/error_exception.php:270" while reading response header from upstream, client: 127.0.0.1, server: localhost, request: "POST /form HTTP/1.1", upstream: "fastcgi://unix:/run/php/php7
```

**Comment by mithaaprilia** [comment]
Same problem with #172

**Comment by lonnieezell** [comment]
Fixed in #308
